proposals/lip-0057.md

```diff
@@ -106,7 +106,7 @@ For the rest of this proposal we define the following constants:
 | `MIN_INIT_ROUNDS` | uint32 | `3` | The minimum number of rounds for the [bootstrap period][lip-0034#bootstrap-period]. |
 | `MAX_NUM_BYTES_Q96` | uint32 | 24 | The maximal number of bytes of a serialized fractional number in Q96 format (see [Reward sharing LIP][lip-0070]). |
 | **Configurable Constants** | | **Mainchain Value** | |
-| `TOKEN_ID_POS` | bytes | `TOKEN_ID_LSK = 0x 00 00 00 00 00 00 00 00` | The [token ID][lip-0051#tokenID] of the token used for staking. |
+| `TOKEN_ID_POS` | bytes | configurable | The [token ID][lip-0051#tokenID] of the token used for staking. |
```

Comment:
If we want to indicate the mainchain value for the configurable constants, I would write here `OWN_CHAIN_ID[0:1] + '00000000000000'`

Reply:
Changed to `OWN_CHAIN_ID[0:1] + '00000000000000'` as it should be a byte constant set in the configuration file and we cannot use a dynamic method call here.